src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.UnitTests/Controllers/ApprenticeshipControllerTests/ApprenticeshipsControllerTests.cs

```diff
@@ -32,58 +35,108 @@ public void Init()
         {
             //Arrange
-            var providerId = (uint) 10;
+            var request = new GetApprenticeshipRequest
+            {
+                ProviderId = 10
+            };
 
             //Act
-            await _controller.GetApprenticeships(providerId);
+            await _controller.GetApprenticeships(request);
 
             //Assert
             _mediator.Verify(m => m.Send(
-                It.Is<GetApprenticeshipsRequest>(r => r.ProviderId.Equals(providerId)), 
+                It.Is<GetApprenticeshipsRequest>(r => 
+                    r.ProviderId.Equals((uint)request.ProviderId)), 
+                It.IsAny<CancellationToken>()), Times.Once);
+        }
+
+        [Test]
+        public async Task GetApprovedApprenticesByPage()
+        {
+            //Arrange
+            var request = new GetApprenticeshipRequest
+            {
+                ProviderId = 10,
+                PageNumber = 4,
+                PageItemCount = 17
+            };
+
+            //Act
+            await _controller.GetApprenticeships(request);
+
+            //Assert
+            _mediator.Verify(m => m.Send(
+                It.Is<GetApprenticeshipsRequest>(r => 
+                    r.ProviderId.Equals((uint)request.ProviderId) &&
+                    r.PageNumber.Equals(request.PageNumber) &&
+                    r.PageItemCount.Equals(request.PageItemCount)), 
                 It.IsAny<CancellationToken>()), Times.Once);
         }
 
         [Test]
         public async Task ReturnApprovedApprentices()
         {
             //Arrange
-            var providerId = (uint)10;
+            var request = new GetApprenticeshipRequest
+            {
+                ProviderId = 10
+            };
+            const int expectedTotalApprenticeshipsFound = 10;
+            const int expectedTotalApprenticeshipsWithAlertsFound = 3;
+
             var expectedApprenticeship = new ApprenticeshipDetails
             {
-                ApprenticeFirstName = "George",
-                ApprenticeLastName = "Test",
+                FirstName = "George",
+                LastName = "Test",
                 Uln = "12345",
                 EmployerName = "Test Corp",
                 CourseName = "Testing Level 1",
-                PlannedStartDate = DateTime.Now.AddDays(2),
-                PlannedEndDateTime = DateTime.Now.AddMonths(2),
-                PaymentStatus = PaymentStatus.Active
+                StartDate = DateTime.Now.AddDays(2),
+                EndDate = DateTime.Now.AddMonths(2),
+                PaymentStatus = PaymentStatus.Active,
+                Alerts = new []{"one", "two"}
             };
 
-            _mediator.Setup(m => m.Send(It.Is<GetApprenticeshipsRequest>(r => r.ProviderId.Equals(providerId)),
-                It.IsAny<CancellationToken>())).ReturnsAsync(new GetApprenticeshipsResponse
+            _mediator.Setup(m => m.Send(It.Is<GetApprenticeshipsRequest>(r => r.ProviderId.Equals((uint)request.ProviderId)),
```

Comment:
We might need to change the logic that requires this uint cast to support a more consistent approach to how we handle provider id values